examples/additive_manufacturing/sintering_physics/inference.py

```diff
@@ -0,0 +1,705 @@
+# ignore_header_test
+# ruff: noqa: E402
+
+# © Copyright 2023 HP Development Company, L.P.
+# SPDX-FileCopyrightText: Copyright (c) 2023 - 2024 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-FileCopyrightText: All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import functools
+import json
+import os
+import tree
+from absl import app, flags
+import numpy as np
+import math
+import pickle
+
+import torch
+
+try:
+    import tensorflow as tf
+except ImportError:
+    raise ImportError(
+        "Mesh Graph Net Datapipe requires the Tensorflow library. Install the "
+        + "package at: https://www.tensorflow.org/install"
+    )
+
+from tqdm import tqdm
+from modulus.models.vfgn.graph_network_modules import LearnedSimulator
+from modulus.datapipes.vfgn import reading_utils
+
+from constants import Constants
+
+FLAGS = Constants()
```

Comment:
@dearleiii  Currently inference code is not working. and There are so many parameters in `constant.py` is not consistent with the `inference.py`.